crypto/merkletrie/cache_test.go

```diff
@@ -391,7 +399,7 @@ func TestCachePageReloading(t *testing.T) {
 
 	earlyCachedNodeCount := mt1.cache.cachedNodeCount
 	// reloading existing cached page multiple time should not cause increase cached node count.
-	page := uint64(mt1.nextNodeID) / uint64(memoryCommitter.GetNodesCountPerPage())
+	page := uint64(mt1.nextNodeID-1) / uint64(defaultTestMemoryConfig.NodesCountPerPage)
```

Comment:
why '-1' ?

Reply:
because the page for nextNodeID might node exists if nextNodeID happen to be a multiple of `NodesCountPerPage`. On the other hand, after we commit, the last added node ( i.e. nextNodeID-1 ) should be stored, and therefore - it's page is there as well.